<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseMenu</class>
 <widget class="QWidget" name="BaseMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#item0,#item1,#item2,#item3,#item4,#item5
{
	border-left:  1px solid qlineargradient(spread:repeat, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 130, 195, 255), stop:0.1 rgba(0, 130, 195, 255), stop:0.333 rgba(255, 255, 255, 255), stop:0.667 rgba(255, 255, 255, 255), stop:0.9 rgba(0, 130, 195, 255), stop:1 rgba(0, 130, 195, 255));
	border-right: 1px solid qlineargradient(spread:repeat, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(0, 130, 195, 255), stop:0.1 rgba(0, 130, 195, 255), stop:0.333 rgba(0, 0, 0, 255), stop:0.667 rgba(0, 0, 0, 255), stop:0.9 rgba(0, 130, 195, 255), stop:1 rgba(0, 130, 195, 255));
	background-color:  qlineargradient(spread:repeat, x1:0.5, y1:0, x2:0.5, y2:1,
stop:0 rgba(0, 130, 195, 255), 
stop:0.666 rgba(0, 130, 195, 255), 
stop:0.667 rgba(0, 0, 0, 255), 
stop:0.718730666 rgba(0, 0, 0, 255),
stop:0.852440666 rgba(0, 130, 195, 255));
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,1,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="item0" native="true">
     <layout class="QVBoxLayout" name="layout0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="item1" native="true">
     <layout class="QVBoxLayout" name="layout1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="item2" native="true">
     <layout class="QVBoxLayout" name="layout2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="item3" native="true">
     <layout class="QVBoxLayout" name="layout3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="item4" native="true">
     <layout class="QVBoxLayout" name="layout4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="item5" native="true">
     <layout class="QVBoxLayout" name="layout5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
